flare/kernels/two_body_mc_simple.py

```diff
@@ -0,0 +1,712 @@
+                             self.cutoff, self.cutoff_func)
+
+    def force_energy(self, env1: AtomicEnvironment, env2: AtomicEnvironment):
+
+        return force_energy(env1.bond_array_2, env1.ctype, env1.etypes,
+                            env2.bond_array_2, env2.ctype, env2.etypes,
+                            self.signal_variance, self.length_scale,
+                            self.cutoff, self.cutoff_func)
+
+    def stress_energy(self, env1: AtomicEnvironment, env2: AtomicEnvironment):
+
+        return stress_energy(env1.bond_array_2, env1.ctype, env1.etypes,
+                             env2.bond_array_2, env2.ctype, env2.etypes,
+                             self.signal_variance, self.length_scale,
+                             self.cutoff, self.cutoff_func)
+
+    def force_force(self, env1: AtomicEnvironment, env2: AtomicEnvironment):
+
+        return force_force(env1.bond_array_2, env1.ctype, env1.etypes,
+                           env2.bond_array_2, env2.ctype, env2.etypes,
+                           self.signal_variance, self.length_scale,
+                           self.cutoff, self.cutoff_func)
+
+    def stress_force(self, env1: AtomicEnvironment, env2: AtomicEnvironment):
+        return stress_force(env1.bond_array_2, env1.ctype, env1.etypes,
+                            env2.bond_array_2, env2.ctype, env2.etypes,
+                            self.signal_variance, self.length_scale,
+                            self.cutoff, self.cutoff_func)
+
+    def stress_stress(self, env1: AtomicEnvironment, env2: AtomicEnvironment):
+        return stress_stress(env1.bond_array_2, env1.ctype, env1.etypes,
+                             env2.bond_array_2, env2.ctype, env2.etypes,
+                             self.signal_variance, self.length_scale,
+                             self.cutoff, self.cutoff_func)
+
+    def force_force_gradient(self, env1: AtomicEnvironment,
+                             env2: AtomicEnvironment):
+        return force_force_gradient(env1.bond_array_2, env1.ctype, env1.etypes,
+                                    env2.bond_array_2, env2.ctype, env2.etypes,
+                                    self.signal_variance, self.length_scale,
+                                    self.cutoff, self.cutoff_func)
+
+    def efs_energy(self, env1: AtomicEnvironment, env2: AtomicEnvironment):
+        return efs_energy(env1.bond_array_2, env1.ctype, env1.etypes,
+                          env2.bond_array_2, env2.ctype, env2.etypes,
+                          self.signal_variance, self.length_scale,
+                          self.cutoff, self.cutoff_func)
+
+    def efs_force(self, env1: AtomicEnvironment, env2: AtomicEnvironment):
+        return efs_force(env1.bond_array_2, env1.ctype, env1.etypes,
+                         env2.bond_array_2, env2.ctype, env2.etypes,
+                         self.signal_variance, self.length_scale,
+                         self.cutoff, self.cutoff_func)
+
```

Comment:
Since all the inputs are the same, perhaps we can define a function like:
```
def get_args(self, env1, env2):
    return (env1.bond_array_2, env1.ctype, env1.etypes,
                          env2.bond_array_2, env2.ctype, env2.etypes,
                          self.signal_variance, self.length_scale,
                          self.cutoff, self.cutoff_func)
```

and then we do 
```
def efs_force(self, env1: AtomicEnvironment, env2: AtomicEnvironment):
    args = self.get_args(env1, env2)
    return efs_force(*args)
```
same for energy_energy, force_force etc.

Reply:
and maybe this get args can be use along with the mask_to_args. would be something like
```
hyp_args = mask_to_args(hyps, hyps_mask)
env_args = get(args, env1, env2)
args = env_args + hyp_args
```

Reply:
Good idea, this cuts down on a lot of unnecessary lines. I added get_args methods to the 2- and 3-body kernels in 5633d98.